fastf1/events.py

```diff
@@ -400,9 +401,12 @@ def get_event(
         force_ergast: [Deprecated, use ``backend='ergast'``] Always use data
             from the ergast database to create the event schedule
 
-        strict_search: Match precisely the query, or default to
+        strict_search: This argument is planned for deprecation, use the
```

Comment:
I think we are deprecating ``strict_search`` right away? Then the argument is planned for removal, but it already is deprecated. So the wording should be changed to "is deprecated".